docs/ar-SA/README.md

```diff
@@ -36,7 +36,6 @@ description: Starship is the minimal, blazing fast, and extremely customizable p
 
```

Comment:
I think it'd be best if we avoid formatting translated files this time around and have Crowdin translate the files with updated formatting.

Reply:
I think this is a good idea. I just caused a lot of merge conflicts for this PR (sorry @chipbuster) by merging the most latest Crowdin PR.

Reply:
@andytom No worries!